src/test/java/uk/nhs/digital/nhsconnect/lab/results/mesh/MeshMailBoxSchedulerTest.java

```diff
@@ -0,0 +1,84 @@
+package uk.nhs.digital.nhsconnect.lab.results.mesh;
+
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.InjectMocks;
+import org.mockito.Mock;
+import org.mockito.junit.jupiter.MockitoExtension;
+import org.springframework.context.ApplicationContext;
+import org.springframework.core.env.Environment;
+import uk.nhs.digital.nhsconnect.lab.results.mesh.scheduler.SchedulerTimestampRepository;
+import uk.nhs.digital.nhsconnect.lab.results.utils.TimestampService;
+
+import java.time.Instant;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.mockito.ArgumentMatchers.anyLong;
+import static org.mockito.ArgumentMatchers.anyString;
+import static org.mockito.ArgumentMatchers.isA;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.when;
+
+@ExtendWith(MockitoExtension.class)
+public class MeshMailBoxSchedulerTest {
```

Comment:
Let's keep this class idiomatic JUnit 5 - no `public` modifiers, here or some of the tests. I find it helps to see at a glance: `public` keywords => JUnit 4; none => JUnit 5.

Applies to some other test classes too.

Reply:
Good point! We're not being consistent here :-)